server/athenian/api/models/web/for_set.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         }
 
         self._repositories = repositories
```

Comment:
Shall we for `repositories` as well? The constructor accepts `None`, but typing is not `Optional`.

Reply:
Repositories may not be empty by the API contract. Those constructor defaults must allow the engine to initialize an object without any kwargs. This is confusing. If I were designing the logic, I would remove all the arguments from `__init__`, but we have to live with what we have.

Reply:
In other words, if `repositories` is None, then the model is not initialized yet. However, `developers` has a default value which is an empty list.